docs(blog): revised maker intro text and links (#185)
spokestack/spokestack-website: content/blog/introducing-spokestack-maker/index.md

description: Spokestack Maker introduces sophisticated AutoML tools for voice interfaces to a new audience of prototypers, enthusiasts, and makers.
tags: News, Product
author: noel

> draft: false

Reviewer: Probably also want to update the date.

hero: introducing-spokestack-maker.png
---

![Introducing Spokestack Maker - AutoML Personal Voice for Creators](introducing-spokestack-maker.png)

Spokestack exists to help put voice into software because voice is a humane way to interface with software and because software needs better, more accessible tools to utilize voice.

[Spokestack Maker](/account/upgrade) introduces sophisticated machine learning tools for voice to a new audience of prototypers, enthusiasts, and makers. Create, train, distribute, and use state-of-the-art personal wake word, keyword, and text-to-speech (TTS) models — with _your_ own voice.

> Voice AI is a difficult field, full of papers with irreproducible results, easy to overlook pitfalls, and undocumented code. You can end up spending all day in Jupyter notebooks babysitting training jobs instead of building your killer app. We know, we’ve done it before! [Spokestack Maker](/account/upgrade) turns voice just another interface developers utilize—like a mouse & keyboard or a touchscreen—to interact with users.

Reviewer: ```suggestion
Voice AI is a difficult field, full of papers with unreproducible results, easy to overlook pitfalls, and undocumented code. You can end up spending all day in Jupyter notebooks babysitting training jobs instead of building your killer app. We know, we’ve done it before! [Spokestack Maker](/account/upgrade) turns voice just another interface developers utilize—like a mouse & keyboard or a touchscreen—to interact with users.
```

Reviewer 2: I'd just change it to "unreproducible"—they're both accepted words as far as I can tell, but the "ir-" prefix does look a little odd. "Inconsistent" doesn't quite fit, as results in papers are a static thing; you just often can't get those same results yourself.

Author: `irreproducible` is a reference to https://en.wikipedia.org/wiki/Journal_of_Irreproducible_Results

> Voice AI is a difficult field, full of papers with irreproducible results, easy to overlook pitfalls, and undocumented code. You can end up spending all day in Jupyter notebooks babysitting training jobs instead of building your killer app. We know, we’ve done it before! [Spokestack Maker](/account/upgrade) turns voice just another interface developers utilize—like a mouse & keyboard or a touchscreen—to interact with users.

Reviewer 2: ```suggestion
Voice AI is a difficult field, full of papers with irreproducible results, hidden pitfalls, and undocumented code. You can end up spending all day in Jupyter notebooks babysitting training jobs instead of building your killer app. We know, we’ve done it before! [Spokestack Maker](/account/upgrade) turns voice into just another interface developers utilize—like a mouse & keyboard or a touchscreen—to interact with users.
```

Spokestack already offers powerful, industry-leading libraries for utilizing voice in [Python](/docs/python/getting-started), [React Native](/docs/react-native/getting-started), [Android](/docs/android/getting-started), and [iOS](/docs/ios/getting-started). With [Spokestack Maker](/account/settings#billing), that voice becomes personal and customizable, utilizing state of the art self-service AutoML technology.

With [Spokestack Maker](/account/upgrade), you have the power of three new tools for training your own [wake word](/features/wake-word), [keyword](/features/keyword), and [TTS](/features/tts) models using data you record yourself. You'll still have access to the same [NLU](/features/nlu) model trainer and TTS showcase available to all existing free accounts.

> Want to test drive powerful machine learning voice features like keyword recognition, wake word activation, and custom AI voices without breaking the bank? [Spokestack Maker](/account/upgrade) brings these features to the creator market that has not benefited from access to these powerful technologies before.

Reviewer: I think there's a couple too many "powerful"s in there. Note the one above as well.

```suggestion
Want to test drive robust machine learning voice features like keyword recognition, wake word activation, and custom AI voices without breaking the bank? [Spokestack Maker](/account/upgrade) brings these features to the creator market that has not benefited from access to these technologies before.
```

Author: i'll go with josh's suggestion, but good catch!

> Want to test drive powerful machine learning voice features like keyword recognition, wake word activation, and custom AI voices without breaking the bank? [Spokestack Maker](/account/upgrade) brings these features to the creator market that has not benefited from access to these powerful technologies before.

Reviewer 2: Agreed, but I'd go a step further:
```suggestion
Want to test drive cutting-edge machine learning features like keyword recognition, wake word activation, and custom AI voices without breaking the bank? [Spokestack Maker](/account/upgrade) brings this enterprise-level technology to the creator market for the first time.
```

> [Spokestack's Maker subscription pricing](/pricing) is tailored for two audiences: hobbyists who want to personalize their projects, and developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality TTS voice.

Reviewer: ```suggestion
[Spokestack's Maker subscription pricing](/pricing) is tailored for two audiences: hobbyists who want to personalize their projects and developers who want to prototype a project as realistically as possible before committing to training a universal wake word model or studio-quality TTS voice.
```
